boards/sparkfun-pro-micro/examples/pro-micro-adc.rs

```diff
@@ -19,7 +20,7 @@ fn main() -> ! {
 
     ufmt::uwriteln!(&mut serial, "Reading analog inputs ...\r").void_unwrap();
 
-    let mut adc = sparkfun_pro_micro::adc::Adc::new(dp.ADC, Default::default());
+    let mut adc = adc::Adc::new(dp.ADC, Default::default());
```

Comment:
Let's keep those changes out of this MR and do them later (if at all) once we have decided what to do with the crate.

Reply:
done